gradle-multi-module/sample-admin/src/main/java/com/sungjun/admin/Application.java

```diff
@@ -0,0 +1,18 @@
+package com.sungjun.admin;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.context.ApplicationContext;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+
+@SpringBootApplication(scanBasePackages = {"com.sungjun", "com.sungjun.dto.*"})
+@EnableJpaRepositories(basePackages = "com.sungjun")
+@EntityScan(basePackages = "com.sungjun")
+public class Application {
```

Comment:
I didn't find the module-info.java file

---
16. Add module-info.java which :
- use interfaces
- requires module with Bank implementation
- requires module with Service implementation
- requires jmp.dto;

Reply:
I had some compilation problems trying to add module-info here. Due to lack of time, i decided to skip this thing.